docker/ray-deps/Dockerfile

```diff
@@ -1,5 +1,7 @@
 ARG BASE_IMAGE=""
-FROM rayproject/base-deps:nightly"$BASE_IMAGE"
+ARG FULL_BASE_IMAGE=rayproject/base-deps:nightly"$BASE_IMAGE"
+FROM "$FULL_BASE_IMAGE"
```

Comment:
I still see `base-deps` -> `ray-deps` -> `ray` -> `ray-ml` structure. How is it changed then?

Reply:
I think you are seeing old code (@can-anyscale , we should remove those old code, and update the docs... it is really confusing for people to leave them in the code base)

the current structure is:

`base -> ray`

or 

`base -> ray-ml-deps -> ray-ml`

we only install the wheel on the last step.

we do not use `ray-deps` any more.

Reply:
This makes the change meaningless so closing.

Reply:
Just for my own understanding you somebody point to the exact docker files?
* base = base-deps?
* where is ray-ml-deps?

I'm checking master branch....

Reply:
Thanks for the detailed explanation! If you can point this doc part I'm happy to change it to help the community :)

Reply:
While I'm in the understanding process of the new way of docker dep tree just wondering how this can work on arm64: https://github.com/ray-project/ray/blob/6360a0673294c2d99309157dc06dd52ff1e8aedf/docker/ray-ml/install-ml-docker-requirements.sh#L59

If this a bug or a feature? 🤔

Reply:
we do not build (or release) `ray-ml` images for arm64.

Reply:
Thanks for the info, that explains huge amount of things.